pkg/csi-common/driver_test.go

```diff
@@ -25,10 +25,32 @@ import (
 	"google.golang.org/grpc/status"
 )
 
-func TestNewFakeCSIDriver(t *testing.T) {
-	// Test New fake driver with invalid arguments.
-	d := NewCSIDriver("", vendorVersion, fakeNodeID)
-	assert.Nil(t, d)
+func TestNewCSIDriver(t *testing.T) {
+	tests := []struct {
+		name   string
+		vendor string
+		nodeID string
+	}{
+		{
+			name:   "",
+			vendor: "0.1.0",
+			nodeID: "fakeID",
+		},
+		{
+			name:   "testDriver",
+			vendor: "0.1.0",
+			nodeID: "",
+		},
+		//{
```

Comment:
remove comments

Reply:
I'll remove the comment for now.

I initially commented it because
https://github.com/kubernetes-sigs/azuredisk-csi-driver/blob/47b827c06326d08d4176fb75e08cd7c2aa0fc093/pkg/csi-common/driver.go#L48

this case is not handled as of now, is there any update as to what I can fix in this?

Reply:
We have no plan to "fix" it, let's remove the comments in test first, thanks.

Reply:
okay